libs/expo/shared/utils/src/lib/html/sanitizeHtmlString.ts

```diff
@@ -0,0 +1,31 @@
+import sanitizeHtml, { IOptions as SanitizeHtmlOptions } from 'sanitize-html';
+
+const defaultSanitizeOptions: SanitizeHtmlOptions = {
+  allowedTags: [], // remove ALL HTML tags
+  allowedAttributes: {}, // remove ALL attributes
+  disallowedTagsMode: 'discard', // completely drop bad tags (like script)
+};
```

Comment:
I'm not sure if i understand these comments

Reply:
not mine. cc: @tglaz